main.tf

```diff
@@ -276,10 +276,12 @@ resource "google_cloudfunctions_function" "function" {
       RESOURCE_TAG_PREFIX        = "${var.cluster_name}"
       PAYG_SCALING_GROUP_NAME    = "${var.cluster_name}-${random_string.random_name_post.result}",
       BYOL_SCALING_GROUP_NAME    = "${var.cluster_name}-${random_string.random_name_post.result}",
+      #TODO: change value toMAIN once core logic is changed.
       MASTER_SCALING_GROUP_NAME  = "${var.cluster_name}-${random_string.random_name_post.result}",
       HEART_BEAT_LOSS_COUNT      = "${var.HEART_BEAT_LOSS_COUNT}",
       SCRIPT_TIMEOUT             = "${var.SCRIPT_TIMEOUT}"
-      MASTER_ELECTION_TIMEOUT    = "${var.MASTER_ELECTION_TIMEOUT}",
+       #TODO: change value toMAIN once core logic is changed.
```

Comment:
to MAIN

also, **MAIN** or **PRIMARY**

Reply:
erased the main comment